File: PHP/Easy/Filters/Sorting/1460__Make_Two_Arrays_Equal_by_Reversing_Subarrays.php
<?php
function canBeEqual($target, $arr): bool
{
    return array_count_values($target) == array_count_values($arr);
}

$target = [1, 2, 3, 4];
$arr = [2, 4, 1, 3];
var_dump(canBeEqual($target, $arr));

File: Python/Easy/Filter/Sorting/1460__Make_Two_Arrays_Equal_by_Reversing_Subarrays.py
from collections import Counter


# def canBeEqual(target: list[int], arr: list[int]) -> bool:
#     target.sort()
#     arr.sort()
#     return target == arr


def canBeEqual(target: list[int], arr: list[int]) -> bool:
    return Counter(target) == Counter(arr)

target = [1, 2, 3, 4]
arr = [2, 4, 1, 3]
print(canBeEqual(target, arr))
